src/core/classes/drop-erc1155-history.ts

```diff
@@ -29,9 +29,17 @@ export class DropErc1155History {
     tokenId: BigNumberish,
   ): Promise<string[]> {
     const a = (await this.events.getEvents("TokensClaimed")).filter((e) =>
-      e.data.tokenId.eq(tokenId),
+      e.data && BigNumber.isBigNumber(e.data.tokenId)
+        ? e.data.tokenId.eq(tokenId)
+        : false,
     );
 
-    return Array.from(new Set(a.map((b) => b.data?.claimer)));
+    return Array.from(
+      new Set(
+        a
+          .filter((b) => typeof b.data?.claimer === "string")
+          .map((b) => b.data.claimer as string),
```

Comment:
yes because I switched it to not return `any` instead `unknown` (because that's what we should be returning, users should have to check the types before they use stuff, we should never return `any` really)